react-app/src/redux/game/actions.js

```diff
@@ -24,6 +32,81 @@ export const updateStatus = status => ({
   status
 });
 
+const updateGameActions = {
+  updateGameSuccess: (data, obj, dispatch) => {
+    const id = data.id;
+    const gameCount = Number(data.gameCount) + 1;
+    let objUpdate = { gameCount };
+    if (obj.winner && obj.winner === USER_PLAYER_MARK) {
+      const points = Number(data.points) + obj.points;
+      const historyPoints = { ...data.historyPoints, [new Date().getTime()]: obj.points };
+      objUpdate = { ...objUpdate, points, historyPoints };
+    }
+    authService.setUser(id, objUpdate);
+    dispatch({
+      type: actionNames.UPDATE_GAME_SUCCESS,
+      loading: false,
```

Comment:
Every action data attribute should be inside `payload`. 
The action object should only have type and payload.
Apply this comment to every action.